src/hdmf/common/resources.py

```diff
@@ -186,88 +196,105 @@ def _add_entity(self, **kwargs):
         resource_entity = Entity(key, resources_idx, entity_id, entity_uri, table=self.entities)
         return resource_entity
 
-    @docval({'name': 'resource', 'type': str, 'doc': 'the name of the ontology resource'},
-            {'name': 'uri', 'type': str, 'doc': 'uri associated with ontology resource'})
+    @docval({'name': 'resource', 'type': str, 'doc': 'The name of the ontology resource.'},
+            {'name': 'uri', 'type': str, 'doc': 'The URI associated with ontology resource.'})
     def _add_resource(self, **kwargs):
         """
-        Add resource name and uri to ResourceTable that will be referenced by the ResourceTable idx.
+        Add resource name and URI to ResourceTable that will be referenced by the ResourceTable idx.
         """
         resource_name = kwargs['resource']
         uri = kwargs['uri']
         resource = Resource(resource_name, uri, table=self.resources)
         return resource
 
     @docval({'name': 'container', 'type': (str, AbstractContainer),
-             'doc': 'the Container/Data object to add or the object_id for the Container/Data object to add'},
-            {'name': 'field', 'type': str, 'doc': 'the field on the Container to add'})
+             'doc': 'The Container/Data object to add or the object id of the Container/Data object to add.'},
+            {'name': 'relative_path', 'type': str, 'doc': 'The relative path on the Container to add.'},
+            {'name': 'field', 'type': str, 'default': '',
```

Comment:
```suggestion
            {'name': 'relative_path', 'type': str, 'doc': 'The relative path on the object to add.'},
            {'name': 'field', 'type': str, 'default': '',
```

We need to be careful here with language. The relative_path is relative with respect to the file hierarchy described by the schema. The Container is an API class, and as such does not have relative paths.